sdk/core/azure-core-http-netty/src/main/java/com/azure/core/http/netty/implementation/NettyUtility.java

```diff
@@ -93,31 +92,35 @@ public static void closeConnection(Connection reactorNettyConnection) {
      * warning will contain the versions found in runtime and the expected versions to be used by the SDK.
      */
     public static void validateNettyVersions() {
```

Comment:
do we have any means to know that it's called once/rare? And that it won't change over time?

Otherwise, I'd at least move initialization of `versionLogInformation` to a static block and made it a static field. Or, as an alternative, have a flag indicating that it's already been logged and not create/log it again.

Reply:
Should only be called once per class loading.